src/pages/Onboard/AcceptInvitationPage.vue

```diff
@@ -0,0 +1,392 @@
+      }
+    },
+    slash1Class() {
+      return {
+        ...this.slashClass,
+        paused: !this.start
+      }
+    },
+    slash2Class() {
+      return {
+        slash: this.$vuetify.breakpoint.smAndUp,
+        'slash-origin-center': this.$vuetify.breakpoint.smAndUp,
+        paused: !this.start
+      }
+    },
+    slash3Class() {
+      return {
+        ...this.slashClass,
+        paused: false
+      }
+    },
+    slash4Class() {
+      return {
+        ...this.slashResourcesClass,
+        'slash-origin-left': this.$vuetify.breakpoint.smAndUp,
+        paused: true
+      }
+    },
+    slash5Class() {
+      return {
+        ...this.slashResourcesClass,
+        'slash-origin-right': this.$vuetify.breakpoint.smAndUp,
+        paused: true
+      }
+    }
+  },
+  mounted() {
+    setTimeout(() => {
+      this.start = true
+    }, 250)
+  },
+  methods: {
+    ...mapActions('tenant', ['setCurrentTenant']),
+    async accept() {
+      this.loading = true
+      const tenant = this.membershipInvitation.tenant
+      const invitationId = this.membershipInvitation.id
+      const accepted = await this.acceptInvitation(invitationId)
+      if (accepted) {
+        sessionStorage.removeItem('invitationId')
+        await this.setCurrentTenant(tenant.slug)
+        this.$router.push({
+          name: 'dashboard',
+          params: { tenant: tenant.slug }
+        })
+      }
+      this.loading = false
+    },
+    async decline() {
+      this.deleting = true
+      const invitationId = this.membershipInvitation.id
+      const declined = await this.declineInvitation(invitationId)
+      if (declined) {
+        this.toDashboard()
+      }
+      this.deleting = false
+    },
+    toDashboard() {
+      sessionStorage.removeItem('invitationId')
+      this.$router.push({ name: 'dashboard' })
+    },
+    async declineInvitation(id) {
+      try {
+        const { data } = await this.$apollo.mutate({
+          mutation: require('@/graphql/Tenant/delete-membership-invitation.gql'),
+          variables: {
+            membershipInvitationId: id
+          }
+        })
+        if (data?.delete_membership_invitation?.success) {
+          return true
+        }
+      } catch (e) {
+        this.mutationErrorMessage = e
+          .toString()
+          .split(':')
+          .pop()
+        this.loading = false
+        this.error = true
+      }
+    },
+    async acceptInvitation(id) {
+      try {
+        const { data } = await this.$apollo.mutate({
+          mutation: require('@/graphql/Tenant/accept-membership-invitation.gql'),
+          variables: {
+            membershipInvitationId: id
+          }
+        })
+        if (data?.accept_membership_invitation?.id) {
+          return true
+        }
+      } catch (e) {
+        this.mutationErrorMessage = e
+          .toString()
+          .split(':')
+          .pop()
+        this.loading = false
+        this.error = true
+      }
+    },
+    slash2Style(i) {
```

Comment:
You shouldn't need the slash style computed props and methods if `accept` is a sub-route of `onboard` - you'll instead want to modify the classes on the parent route (I'd guess this is what's causing the moving slashes)